web/src/components/Upload/UploadValidationResultsTable/UploadValidationResultsTable.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import UploadValidationResultsTable from './UploadValidationResultsTable'
+import { Severity } from './UploadValidationResultsTable'
+
+const meta: Meta<typeof UploadValidationResultsTable> = {
+  component: UploadValidationResultsTable,
+}
+
+export default meta
+
+type Story = StoryObj<typeof UploadValidationResultsTable>
+
+const messages = [
+  {
+    message: 'Upload template version is older than the latest input template',
+    tab: 'Logic',
```

Comment:
I'm wondering if it might be worth making a smol enum for these tabs? I notice we are referencing `logic` and `cover` in `validate-upload` as well so might be nice to have a central place for defs / make them all the same case etc? 🤷‍♀️

Reply:
Good idea - I take it there are bounded values for tab? Is it just those two values? I can make a follow-up PR once confirmed.